build/dsc.rb

```diff
@@ -4,11 +4,15 @@
 require 'pathname'
 require 'erb'
 require 'open-uri'
-require 'mof'
-require 'cim'
 require 'find'
 require 'logger'
-require 'charlock_holmes/string'
+
+# Bundler group 'build'
+if Bundler.rubygems.find_name('mof').any?
```

Comment:
Is the issue that it doesn't work on Windows? Why not shield by `if !Puppet::Util::Platform.windows?` instead?